src/Pages/ProductList/Components/Productslider.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Product from "./Product";
+import Slider from "react-slick";
+import Sliderintro from "./Sliderintro";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import "./Productslider.scss";
+
+class Productslider extends React.Component {
+  state = {
+    cpid: 0,
+  };
+
+  render() {
+    const { products } = this.props;
+    const settings = {
+      infinite: false,
+      speed: 300,
+      arrows: true,
+      slidesToShow: 3,
+      slidesToScroll: 1,
+      variableWidth: true,
+    };
+
+    return (
+      <div>
+        {products.map((el, i) => {
+          console.log(this.state.cpid);
+          return (
+            <div className="Productslider">
+              <section
+                className={
+                  i % 2 === 0 ? "evenSliderContainer" : "oddSliderContainer"
+                }
```

Comment:
지금 방식으로도 상관 없지만, 자바스크립트에서 0은 falsy한 값으로 인식한다는 점을 이용하면 다음과 같이 할 수도 있겠습니다.
```suggestion
                className={
                  i % 2 ? "oddSliderContainer" : "evenSliderContainer"
                }
```